tests/test_spectral_indices.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import Mock
+from openeo.extra.spectral_indices.spectral_indices import _callback, _get_expression_map
+from processes import array_create
+
+x = array_create([9,4,6,1,3,8,2,5,7,11,7,2])
+cube = Mock()
```

Comment:
it should not be necessary to mock a cube.


In test_datacube100.py you can find loads of examples where a datacube is constructed from a mock collection, but the datacube is a real datacube.